src/backend/catalog/pipeline_query.c

```diff
@@ -916,6 +938,8 @@ DefineContinuousTransform(Oid relid, Query *query, Oid typoid, Oid osrelid, Oid
 	values[Anum_pipeline_query_seqrelid - 1] = ObjectIdGetDatum(InvalidOid);
 	values[Anum_pipeline_query_sw_attno - 1] = ObjectIdGetDatum(InvalidOid);
```

Comment:
We should change this to `int16` as well and use `InvalidAttributeNumber` instead.